config/manager/pdb.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: controller-manager
+  namespace: system
```

Comment:
Is this the right namespace? 
Perhaps omit namespace and let kustomize set it?

Off topic, but it would be nice to move this into the fr-system namespace ;-)

Reply:
that ns is set to `system` just so that it matches all other manifests in the project. The namespace is set to `secret-agent-system` by the top level kustomization file.